components/SideBar.vue

```diff
@@ -1,21 +1,51 @@
 <template>
   <div class="sidebar grey-100 d-flex flex-column">
-    <header-bar pad-bottom/>
-    <simplebar class="narrow-scrollbar flex-grow-1">
-      <roster-list
-          :pinned="[]"
-          :items="items"
-          :selected-jid="selectedJid"/>
-    </simplebar>
+    <header-bar/>
+    <div class="d-flex flex-grow-1 flex-column">
+        <accordion-item header="Roster"
+                        :expanded="!expanded"

```

Comment:
There's probably a better way about this than a bunch of NOTs. I mean we should just number the things and check against the number, if we ever end up with more accordion items.

Reply:
Ideally, the items would be generated with a `<template v-for="(rib, i) in ribs"/>` and dynamic components. Then the number thing could work, i.e. `:expanded="expanded === i"`